gusto/timestepping/timestepper.py

```diff
@@ -180,7 +180,7 @@ def log_field_stats(self):
                 max_val = field_data.max()
                 logger.debug(f'{field_name}, min: {min_val}, max: {max_val}')
 
-    def run(self, t, tmax, pick_up=False):
+    def run(self, t, tmax, pick_up=False, first_run=True):
```

Comment:
I know it's not really my place to review Gusto code but I feel this might be more elegant if this were called `init_io` or similar. Or you could even set this attribute inside `Timestepper.__init__`.

Reply:
yes, this is a nice idea @connorjward  - thanks! your comments are always appreciated!